scipy/stats/_bootstrap.py

```diff
@@ -208,6 +225,9 @@ def bootstrap(data, statistic, *, vectorized=True, axis=0,
         If `vectorized` is set ``False``, `statistic` will not be passed
         keyword argument `axis`, and is assumed to calculate the statistic
         only for 1D samples.
+    paired : bool, optional
```

Comment:
Can be done later, but if for some parameters you put the default value close to the type, I would do it for all to have the same style along the file at least.